docs/usage.md

```diff
@@ -6,61 +6,143 @@
 
 ## Introduction
 
-<!-- TODO nf-core: Add documentation about anything specific to running your pipeline. For general topics, please point to (and add to) the main nf-core website. -->
+The oncoanalyser pipeline typically runs from FASTQs or BAMs and supports two modes: (1) whole genome and/or
+transcriptome, and (2) targeted panel. Launching an analysis requires only the creation of a samplesheet that describes
+details of each input such as the sample type (tumor or normal), sequence type (DNA or RNA), and filepath.
 
-## Samplesheet input
+Various aspects of an oncoanalyser analysis can be configured to fit a range of needs, and many of these are considered
+[advanced usage](#advanced-usage) of the pipeline. The most useful include:
 
-You will need to create a samplesheet with information about the samples you would like to analyse before running the pipeline. Use this parameter to specify its location. It has to be a comma-separated file with 3 columns, and a header row as shown in the examples below.
+- precise process selection
+- starting from existing data
+- granular control over reference/resource files
 
-```bash
---input '[path to samplesheet file]'
+These features enable oncoanalyser to be run in a highly flexible way. For example, an analysis can be run with existing
+PURPLE data as the starting point and skip variant calling processes. Additionally, reference/resource files can be
+staged locally to optimise execution or modified to create user-defined driver gene panels.
+
+> [!WARNING]
+> There are important requirements when using BAMs as input instead of FASTQs:
+>
+> - STAR must have been run with [specific
+>   parameters](https://github.com/hartwigmedical/hmftools/tree/master/isofox#a-note-on-alignment-and-multi-mapping),
+>   this is critical for WTS data, and
+> - reads are expected to have been aligned to one of the Hartwig-distributed reference genomes (user-defined genomes may be used though are not recommended)
```

Comment:
Could add a link here to a page outlining which files/params would need to be changed in order run the pipeline from a user-defined genome